docs/src/pages/index.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import clsx from "clsx";
+import Layout from "@theme/Layout";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import styles from "./styles.module.css";
+
+const features = [
+  {
+    title: <>Run a Validator</>,
+    imageUrl: "docs/running-validator/README",
+    description: <>Learn how to start a validator on the Solana cluster.</>,
+  },
+  {
+    title: <>Launch an Application</>,
+    imageUrl: "docs/apps/README",
+    description: <>Build superfast applications with one API.</>,
+  },
+  {
+    title: <>Participate in Tour de SOL</>,
+    imageUrl: "docs/tour-de-sol/README",
+    description: (
+      <>
+        Participate in our incentivised testnet and earn rewards by finding
+        bugs.
+      </>
+    ),
+  },
+  {
+    title: <>Integrate the SOL token into your Exchange</>,
+    imageUrl: "docs/integrations/exchange",
+    description: (
+      <>
+        Follow our extensive integration guide to ensure a seamless user
+        experience.
+      </>
+    ),
+  },
+  {
+    title: <>Create or Configure a Solana Wallet</>,
+    imageUrl: "docs/wallet-guide/README",
+    description: (
+      <>
+        Whether you need to create a wallet, check the balance of your funds, or
+        take a look at what's out there for housing SOL tokens, start here.
+      </>
+    ),
+  },
+  {
+    title: <>Learn About Solana's Architecture</>,
+    imageUrl: "docs/cluster/README",
+    description: (
+      <>
+        Familiarize yourself with the high level architecture of a Solana
+        cluster.
+      </>
+    ),
+  }, //
+  // {
+  //   title: <>Understand Our Economic Design</>,
```

Comment:
Why is this one commented out?

Reply:
To keep visual consistency — it'd mess up the 3x2 table layout.